pkg/api/scylla/v1alpha1/types_operatorconfig.go

```diff
@@ -25,6 +25,10 @@ type ScyllaOperatorConfigSpec struct {
 	// Setting this field renders your cluster unsupported. Use at your own risk.
 	// +optional
 	UnsupportedPrometheusVersionOverride *string `json:"unsupportedPrometheusVersionOverride,omitempty"`
+
+	// configuredClusterDomain allows users to set the configured Kubernetes cluster domain explicitly, instead of letting the Scylla Operator to automatically discover it.
```

Comment:
```suggestion
	// configuredClusterDomain allows users to set the configured Kubernetes cluster domain explicitly, instead of letting the Scylla Operator automatically discover it.
```

Reply:
done

Reply:
it still says `the Scylla Operator` - you shouldn't use articles before proper nouns, but I won't hold merging because of that

Reply:
Sorry missed that, fixed.